src/java.base/share/classes/java/util/concurrent/ConcurrentHashMap.java

```diff
@@ -974,12 +987,12 @@ public boolean containsKey(Object key) {
      *         specified value
      * @throws NullPointerException if the specified value is null
      */
+    @Override
     public boolean containsValue(Object value) {
-        if (value == null)
-            throw new NullPointerException();
+        Objects.requireNonNull(value);
         Node<K,V>[] t;
         if ((t = table) != null) {
-            Traverser<K,V> it = new Traverser<K,V>(t, t.length, 0, t.length);
+            Traverser<K,V> it = new Traverser<>(t, t.length, 0, t.length);
```

Comment:
Using diamond is one way to modernize this code, using `var` would be another, even more concise way. Consider: 
```suggestion
            var it = new Traverser<K, V>(t, t.length, 0, t.length);
```
I don't support blindly using `var` everywhere, but I believe when the RHS of the assignment spells out the type that's exactly the kind of usage envisioned for it.